### PR TITLE
fix: pass on the auth from the download URL

### PR DESCRIPTION
**Pull request by OrKoN**
Updated node types but the method should be available from Node 15.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -129,7 +129,7 @@
     "@types/diff": "5.0.2",
     "@types/mime": "3.0.1",
     "@types/mocha": "10.0.1",
-    "@types/node": "18.14.6",
+    "@types/node": "20.2.5",
```
Comment (Lightning00Blade) [inline package.json:132]: Why is this needed? The export the change needs was added in Node 15.

Comment (OrKoN) [reply package.json:132]: Not in the types package.

**Review by Lightning00Blade — APPROVED** [review]
